### ISSUE TITLE
`invest` commands should launch the workbench

### ISSUE THREAD

**Issue by emlys** [natcap/invest#1106]
Before deprecating the QT interface, we should update the CLI to launch the workbench:
- Open the workbench home tab on `invest launch`
- Open the appropriate model tab on `invest run model`

**Comment by davemfish** [comment]
This might be really hard! Since the workbench isn't running in a python environment.

This might be one thing we lose by moving away from the all-python ecosystem.

**Comment by emlys** [comment]
You're right, the `invest` tool that we distribute with the python package won't be able to launch the UI any more. I guess I'd like to have an equivalent command available for development. Something like
`invest-ui launch` that would `cd workbench && yarn start`. I'm not sure how to programmatically open a model tab though.

**Comment by dcdenu4** [comment]
I'll second that this would be handy to have for development. I'm working on something in HQ and it's been convenient to iterate and debug using the UI with `invest run habitat_quality`. Of course I could just run a script as well.
